Bottom: 012555c5d78eebf557839933360fcbfc00fe77a2
Top:    afe42c06959b04243ca2eed6bcd95e1757d27460
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-05-06 17:01:46 +0100

Turn on thindraw poly eye-candy


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 51295d7..0750d0f 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -904,7 +904,7 @@ ghid_thindraw_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box)
 
   double old_alpha_mult = gtk_gc->alpha_mult;
   common_thindraw_pcb_polygon (gc, poly, clip_box);
-#if 0
+#if 1
   ghid_set_alpha_mult (gc, gtk_gc->alpha_mult * 0.25);
   hid_draw_fill_pcb_polygon (gc, poly, clip_box);
   ghid_set_alpha_mult (gc, old_alpha_mult);
